lib/nerves_ssh.ex

```diff
@@ -13,6 +13,14 @@ defmodule NervesSSH do
   # some time to recover.
   @cool_off_time 500
 
+  @typedoc """
+  A name used to refer to a NervesSSH-managed SSH daemon
+
+  This is only needed when running multiple daemons. The default is `NervesSSH`.
+  """
+  @type name() :: any()
```

Comment:
```suggestion
  @type name() :: GenServer.name()
```

Reply:
That's actually not true since this uses `Registry` to track processes. I wasn't sure why it did that, though.

Reply:
O, you're right. I had forgotten we did that (and also why?)